x-pack/plugins/snapshot_restore/public/application/sections/home/repository_list/repository_details/repository_details.tsx

```diff
@@ -356,21 +356,16 @@ export const RepositoryDetails: React.FunctionComponent<Props> = ({
               </EuiCodeBlock>
             </div>
           ) : (
-            <EuiCallOut
-              color="danger"
-              iconType="alert"
-              title={i18n.translate('xpack.snapshotRestore.repositoryDetails.cleanupErrorTitle', {
-                defaultMessage: 'Sorry, there was an error cleaning the repository.',
-              })}
-            >
-              <p>
-                {cleanup.error
-                  ? JSON.stringify(cleanup.error)
-                  : i18n.translate('xpack.snapshotRestore.repositoryDetails.cleanupUnknownError', {
-                      defaultMessage: '503: Unknown error',
```

Comment:
I'm missing the background on this particular error scenario. @cuff-links Do you recall any details around this or how we could reproduce?